src/lib/core/typography/_typography.scss

```diff
@@ -42,17 +42,21 @@
 }
 
 // Adds the base typography styles, based on a config.
-@mixin mat-typography($config: mat-typography-config(), $selector: '.mat-typography') {
+// TODO(crisbeto): the headers need to handle margins as well.
+@mixin mat-base-typography($config, $selector: '.mat-typography') {
   .mat-h0, .mat-hero-header {
     @include mat-typography-level-to-styles($config, display-4);
+    letter-spacing: -0.05em;
```

Comment:
I vote for undoing this and just using default spacing, some things look kind of cramped when using these settings, for example the page header.

Page header before this PR:
![old-title](https://cloud.githubusercontent.com/assets/14793288/25728350/4bc670bc-30e3-11e7-9d4a-5c80827b4971.png)

Page header after this PR:
![new-title](https://cloud.githubusercontent.com/assets/14793288/25728353/4f0d5754-30e3-11e7-8134-cc8dfc58cdc0.png)